ando/engine.py

```diff
@@ -234,26 +234,26 @@ def is_AnDO_verbose(directory):
 
 def is_AnDO_verbose_Format(names):
     """
-    Check if file path adheres to AnDO.
+    Check if file path adhere to AnDO.
     Main method of the validator. uses other class methods for checking
     different aspects of the directory path.
 
     Args:
         names ([list]): [names to check]
 
     Raises:
-        ExperimentError: raised if it doesn't the experiment rules
-        SessionError: raised if it doesn't the session rules
-        SubjectError: raised if it doesn't the subject rules
-        SourceError: raised if it doesn't the source rules
-        RawDataError: raised if it doesn't the rawdata rules
-        DerivativeDataError: raised if it doesn't the derivatives rules
-        MetaDataError: raised if it doesn't the metadata rules
-        SourceNotFound: raised if it doesn't the source rules
+        ExperimentError: raised if it does not respect respect respect respect respect the experiment rules
```

Comment:
this is a lot of respect ;)
and below also, there is some double respect!

Reply:
Hum I think there is a little bug I don't have it on my file 
![image](https://user-images.githubusercontent.com/17256769/92506021-1b008a80-f205-11ea-903b-63ba4a170b17.png)